pkg/docker/config/config.go

```diff
@@ -183,7 +190,22 @@ func RemoveAllAuthentication(sys *types.SystemContext) error {
 			}
 			logrus.Debugf("error removing credentials from kernel keyring")
 		}
+		if len(auths.CredHelpers) != 0 {
+			for cred, registry := range auths.CredHelpers {
+				if err := deleteAuthFromCredHelper(cred, registry); err != nil {
+					logrus.Debugf("error removing credentials for %s from credential helper %s: %q", registry, cred, err)
+					return false, err
```

Comment:
Should we ignore registry does not exist errors?  Could we hit a race where two apps are removing the creds at the same time.  Do we want to  save the error and continue deleting other registries.

Reply:
fix this with ignoring IsErrCredentialsNotFoundMessage.